conformance/upload_e2e.py

```diff
@@ -1,328 +0,0 @@
-    proc.communicate()
-    if proc.returncode != 0:
-        raise RuntimeError(
-            'Failed to upload with exit code: %d' % proc.returncode)
-
-
-def get_current_account(dry_run):
-    """gets the currently active gcp account by shelling out to gcloud"""
-    cmd = ['gcloud', 'auth', 'list',
-           '--filter=status:ACTIVE', '--format=value(account)']
-    print('Run:', cmd, file=sys.stderr)
-    if dry_run:
-        return ""
-    return subprocess.check_output(cmd, encoding='utf-8').strip('\n')
-
-
-def set_current_account(account, dry_run):
-    """sets the currently active gcp account by shelling out to gcloud"""
-    cmd = ['gcloud', 'config', 'set', 'core/account', account]
-    print('Run:', cmd, file=sys.stderr)
-    if dry_run:
-        return None
-    return subprocess.check_call(cmd)
-
-
-def activate_service_account(key_file, dry_run):
-    """activates a gcp service account by shelling out to gcloud"""
-    cmd = ['gcloud', 'auth', 'activate-service-account', '--key-file='+key_file]
-    print('Run:', cmd, file=sys.stderr)
-    if dry_run:
-        return
-    subprocess.check_call(cmd)
-
-
-def revoke_current_account(dry_run):
-    """logs out of the currently active gcp account by shelling out to gcloud"""
-    cmd = ['gcloud', 'auth', 'revoke']
-    print('Run:', cmd, file=sys.stderr)
-    if dry_run:
-        return None
-    return subprocess.check_call(cmd)
-
-
-def parse_args(cli_args=None):
-    if cli_args is None:
-        cli_args = sys.argv[1:]
-    parser = argparse.ArgumentParser()
-    parser.add_argument(
-        '--bucket',
-        help=('GCS bucket to upload the results to,'
-              ' of the form \'gs://foo/bar\''),
-        required=True,
-    )
-    parser.add_argument(
-        '--year',
-        help=('the year in which the log is from, defaults to the current year.'
-              ' format: YYYY'),
-        default=str(datetime.datetime.now().year),
-    )
-    parser.add_argument(
-        '--junit',
-        help='path or glob expression to the junit xml results file(s)',
-        required=True,
-    )
-    parser.add_argument(
-        '--log',
-        help='path to the test log file, should contain the ginkgo output',
-        required=True,
-    )
-    parser.add_argument(
-        '--dry-run',
-        help='if set, do not actually upload anything, only print actions',
-        required=False,
-        action='store_true',
-    )
-    parser.add_argument(
-        '--metadata',
-        help='dictionary of additional key-value pairs that can be displayed to the user.',
-        required=False,
-        default=str(),
-    )
-    parser.add_argument(
-        '--key-file',
-        help='path to GCP service account key file, which will be activated before '
-        'uploading if provided, the account will be revoked and the active account reset '
-        'on exit',
-        required=False,
-    )
-    return parser.parse_args(args=cli_args)
-
-
-def main(cli_args):
```

Comment:
This is potentially a useful way for people to figure out how to upload job results when not using prow.

Reply:
thanks for the heads up -- as long as this continues to live on in test-infra for existing users everything should be fine.

we might want to bring it or something similar back here for future users though 😅